x-pack/plugins/siem/public/store/timeline/helpers.ts

```diff
@@ -207,6 +209,13 @@ export const applyDeltaToCurrentWidth = ({
   };
 };
 
+const queryMatchCustomizer = (dp1: QueryMatch, dp2: QueryMatch) => {
```

Comment:
nit: Unit tests for this if you want

Reply:
it is explecit tested with the addition of unit test